docs/src/3.2.Camera.mdx

```diff
@@ -24,6 +24,9 @@ You can move the camera around using the mouse/trackpad, or by using the keyboar
 | `distance`          | `number`  | `75`           | distance between the camera and target                                                          |
 | `phi`               | `number`  | `Math.PI/4`    | angle in radians from +z pole to -z pole, range: [0, π]. Ignored when `perspective` is false    |
 | `thetaOffset`       | `number`  | `0`            | offset azimuth angle in radians, +θ corresponds to +x→+y, range: [0, 2π]                        |
+| `fov`               | `number`  | `Math.PI/4`    | field of view in radians                                                                        |
+| `nearFrustum`       | `number`  | `0.01`         | distance to the near frustum                                                                    |
```

Comment:
Re naming: I think "frustum" refers to the whole viewable box, not the near/far values themselves. These are called near/far _planes_, but this is pretty standard, so maybe we can just call them `near` and `far`? Also it looks like `fovy` is the standard name for fov, with `y` indicating that it's in the vertical direction.

references:
https://www.khronos.org/registry/OpenGL-Refpages/gl2.1/xhtml/gluPerspective.xml (uses fovy/zNear/zFar)
https://www.khronos.org/registry/OpenGL-Refpages/gl2.1/xhtml/glFrustum.xml (uses nearVal/farVal)
http://glmatrix.net/docs/module-mat4.html#.perspective (uses fovy/near/far)

Reply:
👍️ Fixed